terra-jupyter-gatk-ovtf/gatk-ovtf.patch

```diff
@@ -1,8 +1,9 @@
-From a22f79681d517ba2c6a7677638aab96c09a5aace Mon Sep 17 00:00:00 2001
```

Comment:
is this file intentionally source controlled? Seems to be the output of a `git diff`

Reply:
curious as to what this file is for as well :)

Reply:
I asked the same thing. :) I think it's a git patch for making gatk work with the latest `tensorflow`. In fact we might do similar in `terra-jupyter-gatk` to fix the issue with `CNNScoreVariants` (see convo in https://github.com/DataBiosphere/terra-docker/pull/256).